src/index.js

```diff
@@ -2,5 +2,16 @@ import dotenv from 'dotenv';
 import app from './app';
 
 dotenv.config();
+app.use((req, res) => {
```

Comment:
@Niyonsengaeric ,
I think this should be moved into src/app.js where we configured and exported express app

Reply:
Moved 👍